### PR TITLE
Dan, I added some code

### PR DESCRIPTION
**Pull request by samuelcrosland**

### CODE REVIEW

**Comment by danhodkinson** [comment]
no description. try harder

**Comment by samuelcrosland** [comment]
How do I even add a description?

**Comment by samuelcrosland** [comment]
@danhodkinson ???!?

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="The website for Samuel Crosland. A designer and product manager."
     <meta name="author" content="Samuel Crosland">
+  <!-- DAN I ADDED SOME CODE!-->
```
Comment (danhodkinson) [inline index.html]: i can't let you merge in code like this

**Review by danhodkinson — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (samuelcrosland) [reply index.html]: Hmm. Ok. 
Lemme change it

**Review by danhodkinson — APPROVED** [review]

**Pull request merged**